dev-python/jinja2/jinja2-3.1.6.recipe

```diff
@@ -36,9 +36,10 @@ for pythonVersion in ${PYTHON_VERSIONS[@]}; do
 		markupsafe_$pythonPackage
 		cmd:python$pythonVersion
 		\""
-	REPLACES="
+	eval "REPLACES_$pythonPackage=\"
+		jinja
```

Comment:
This one would also fit the general `REPLACES` without a suffix, but I guess it doesn't really matter because it is empty anyway.

Reply:
Ah right, could move it out of the eval here, wanted to get both out of the way, your solution sounds better.